Fabric.Terminology.API/Scripts/Terminology-Intall-Utilities.psm1

```diff
@@ -1,28 +1,130 @@
-Class TerminologyConfig {
-    [String]$siteName
-    [String]$appName
-    [String]$sqlServerAddress
-    [String]$appPool
-    [String]$applicationEndpoint
-    [String]$discoveryServiceUrl
-    [PSCredential]$iisUserCredentials
+function Get-UserValueOrDefault {
+    param(
+        [String] $Default,
+        [String] $Prompt
+    )
+
+    $value = Read-Host "$Prompt or hit enter to accept the default value [$Default]"
+
+    if ([string]::IsNullOrWhiteSpace($value)) {
+        return $Default
+    }
+    else {
+        return $value
+    }
+}
+
+function Get-ConfigValue {
```

Comment:
I'm surprised this isn't a function in the DosInstallUtilities. Does it make sense to put it there?

Reply:
I had a conversation with Brett and Steve and they will evaluate what we created and pull in the things that makes sense